templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml

```diff
@@ -27,6 +27,12 @@ Parameters:
     Description: CloudTrail name
     Type: String
 
+  pDelegatedAdminAccountId:
```

Comment:
this will be log archival account id right ?

Reply:
No, Delegated administrator account(Audit Account) has administrative permissions on all trails and event data stores in the organization